examples/globe.js

```diff
@@ -3,26 +3,52 @@
 
 // Define initial camera position
 var positionOnGlobe = { longitude: 2.351323, latitude: 48.856712, altitude: 25000000 };
+var promises = [];
 
 // `viewerDiv` will contain iTowns' rendering area (`<canvas>`)
 var viewerDiv = document.getElementById('viewerDiv');
+var miniDiv = document.getElementById('miniDiv');
 
 // Instanciate iTowns GlobeView*
 var globeView = new itowns.GlobeView(viewerDiv, positionOnGlobe, { renderer: renderer });
+var miniView = new itowns.GlobeView(miniDiv, positionOnGlobe, {
+    maxSubdivisionLevel: 2,
+    noControls: true,
+    alphaClearValue: 0,
+});
 
-var promises = [];
+var minDistance = 10000000;
+var maxDistance = 30000000;
+
+miniView.mainLoop.gfxEngine.renderer.setClearColor(0xff0000, 0.0);
+
+globeView.onAfterRender = function onAfterRender() {
+    var distanceCamera = globeView.camera.camera3D.position.length();
+    var distance = Math.min(Math.max(distanceCamera * 1.5, minDistance), maxDistance);
+    var camera = miniView.camera.camera3D;
+    camera.position.copy(globeView.controls.moveTarget()).setLength(distance);
+    camera.lookAt(globeView.controls.moveTarget());
+    miniView.notifyChange(true);
+};
 
 function addLayerCb(layer) {
     return globeView.addLayer(layer);
 }
+
+function addLayerCbM(layer) {
```

Comment:
Suggestion: maybe than adding a duplicate function we could use the following syntax instead:
```js
promises.push(itowns.Fetcher.json('./layers/JSONLayers/Ortho.json').then(globeView.addLayer.bind(globeView)));
```
WDYT?